### ISSUE TITLE
Assessments

### ISSUE THREAD

**Issue by nickmcintyre** [JithinKS97/dynamic-learning#50]
What types of assessments do y'all want to prioritize? [Perseus](https://github.com/khan/perseus) could be a source of inspiration.

@JithinKS97 had an interesting idea in #39 of having students input a result based on interaction with a simulation. @ashneeldas2 started work on a dedicated assessment section; perhaps we can embed them in Workbooks along with simulations.

**Comment by JithinKS97** [comment]
Yeah. May be it can be implemented by creating a new component (Maybe something like Assessment box) which can be made part of the object in slides array.

**Comment by ashneeldas2** [comment]
I think the best way to do this would be to have various different components for different question types. I have started working on a MultipleChoice component which will function similarly to the TextBox component.

**Comment by nickmcintyre** [comment]
@ashneeldas2 the youcubed group has some interesting [research on assessment and grading](https://www.youcubed.org/resource/assessment-grading/). A few ideas stand out:

- Encourage/streamline feedback to drive assessment for learning
- Grade on a 4-point scale where A=4 and F=0
- Enable anonymous mode for some discussions
- Make drawings an assessment type
- Allow re-submission of workbooks
